configure.ac

```diff
@@ -17,6 +17,10 @@ AC_PROG_LN_S
 AC_PROG_MAKE_SET
 AC_PROG_RANLIB
 AC_PROG_YACC
+AC_PATH_PROG(BISON, bison)
```

Comment:
This change should _not_ get into develop/master as Bison is a maintainer-only dependency - and common users should just install from the tarball.

The header change is totally correct and missing though, I'll create a PR for this targeting develop.

Note: Tt is quite likely that _back then_ Bison was always needed, that is definitely not the case any more in the current "develop" branch.

Reply:
I see. Thank you.